templates/post/post.css

```diff
@@ -25,31 +30,36 @@
 /* Post Hero  */
 @media (min-width: 600px) {
   .post main {
-    column-gap: 1.5em;
     display: grid;
-    grid-template-areas:
-    "hero hero"
-    "sidebar content"
-    "sidebar related-content"
-    "sidebar post-cards";
+    column-gap:1rem;
     grid-template-columns: 1fr 3fr;
     margin: 2rem auto;
     max-width: var(--section-max-width);
+    padding: unset;
   }
 
-  .post main .section > div.hero-wrapper {
+  .post main .hero picture {
+    z-index:1;
+    padding-bottom:unset;
+  }
+
+  .post main .section:not(.hero-container) {
+    padding: unset;
+  }
+
+  .post .hero-wrapper {
     margin:0;
   }
 
-  .post main > div.hero-container {
-    grid-area: hero;
+  .post .hero-container {
+    grid-column: 1 / -1;
     padding-bottom:4em;
   }
 }
 
 /* grid post sections */
 .post .section:nth-of-type(2) {
```

Comment:
I really despise nth of type selectors unless absolutely necessary, suggestion on how to do this instead:
```
/* set all sections to be columns 2, except the sidebar and the hero */
.post .section:not(.hero-container, .post-sidebar) {
grid-column: 2;
}
.post .section .default-content-wrapper h3 {
/* h3 style go here */
}
```
I'm also curious about those h3 styles. do we need similar styles for h2/h4/h5/h6?